src/openpolicyagent/policies/admin.rego

```diff
@@ -14,4 +14,12 @@ allow {
     api.method = "GET"
     api.path = ["api", "v1", "admin", "clusters"]
     api.token.type = "user"
-}
\ No newline at end of file
+}
+
+# Allow POST access to /api/v1/admin/clusters for admin
+allow {
+    api.method = "POST"
+    api.path = ["api", "v1", "admin", "clusters"]
+    api.token.type = "user"
+    api.token.user.id = "00000000-0000-0000-0000-000000000000"
```

Comment:
Lets keep this aligned with the rule above.

Reply:
It's designed to be like this.
`GET` is open to all users
`POST `  is only open to admin

Reply:
Sorry, my bad. Makes perfect sense that way, of course.